src/test/utils/blobops.js

```diff
@@ -5,7 +5,7 @@ const P = require('../../util/promise');
 const azure_storage = require('../../util/azure_storage_wrap');
 const RandStream = require('../../util/rand_stream');
 
-
+//TODO:: Remove and use cloud_functions::getAzureConnection ??
```

Comment:
The way we get the credential now in the k8s tests env is a secret that translate to env variables. this apply to AZURE and AWS.

So we should add the secret for travis.

Reply:
See my addition in cloud_functions

Reply:
yes, i saw. it still apply. 
it is more a general comment, using cloud credentials should be set as a secret in the test namespace.